### ISSUE TITLE
Extra Empty Lines Cause Tables to Break

### ISSUE THREAD

**Issue by kookma** [flibbles/tw5-uglify#14]
Using TiddlyWIki 5.3.6-pre and uglifying some kookma plugins result in breaking tables. For example, in Shiraz plugin a dynamic table is creates as below

```
<div data-bs-theme=<<color-scheme>> class="tbl-container" style="max-width:100%;">
	<table class=<<class>> style="caption-side:top">
		<!-- reveal caption and edit button -------------------->
		<caption style="text-align:left;">
			<%if [<editButton>match[yes]] %>
...
```

After uglifying, the tiddler looks like below

```
<div data-bs-theme=<<color-scheme>>class=tbl-container style=max-width:100%;>
	<table class=<<class>>style=caption-side:top>
		
		<caption style=text-align:left;>
			<%if [<editButton>match[yes]]%>
``` 

NOTE to the empty line created by Uglify (after deleteing the comment line). This will render by TiddlyWiki with extra >p> tag, e.g

```
<div data-bs-theme=<<color-scheme>>class=tbl-container style=max-width:100%;>
	<table class="" style=caption-side:top>
        <p>    <---------------------------------------------------------------------------
```

and this breaks the table.

**Expected behavior**
When uglify remove a comment line, it should not create empty line

**Comment by flibbles** [comment]
Sure, I can probably have this fixed by tonight or this weekend.

But so you know, this problem wouldn't occur if those plugins used `\whitespace trim`. It'll result in much better compression as well.

Edit: Not to say that the problem is on your end. It's definitely a bug with Uglify, but in general, [you get the most (and safest) results out of Uglify the more you use `\whitespace trim`](https://flibbles.github.io/tw5-uglify/#Trimming%20Whitespace).

**Comment by flibbles** [comment]
Should be fixed with v1.9.2.

**Comment by kookma** [comment]
Tested with TiddlyWiki 5.3.6pre and it works like a charm.
Thank you!
